src/dsp_tools/commands/excel2json/models/input_error.py

```diff
@@ -240,3 +241,44 @@ def execute_error_protocol(self) -> str:
         if self.message_path:
             msg.append(f"The error occurred at {self.message_path}")
         return separator.join(msg)
+
+
+@dataclass(frozen=True)
+class ListNodeProblem:
+    node_id: str
+    problems: dict[str, str]
+
+    def execute_error_protocol(self) -> str:
+        msg = [
```

Comment:
If the `problems` dict has no entries, this will generate an ugly message. Perhaps you should check that before.

Reply:
For the excel and the Node, problems is never empty, but it is true that List itself may not have any problems.